### ISSUE TITLE
Unable to Edit Loops

### ISSUE THREAD

**Issue by tatumrae** [dudaster/ele-custom-skin#146]
I'm not able to edit the Loop posts types, either with WP's "Edit" or "Edit with Elementor". When I click "Edit", I get an error message "Sorry, you are not allowed to edit this item."

If I try to use "Edit With Elementor", it just redirects to the edit.php page (Posts). I am also still able to edit other Theme Builder elements like Header, Archive and Footer.

I've narrowed it down to a conflict with another plugin called "Tevolution" but unfortunately, we need this plugin in order to access our Custom Fields. 

Is there a way to fix this without going through the other plugin developer?

**Comment by dudaster** [comment]
What exactly are you trying to edit?

**Comment by tatumrae** [comment]
The loop. I’m just trying to create the custom skin.

**Comment by tatumrae** [comment]
Is there any update for troubleshooting this issue? I'm still not able to edit loops (create a custom skin) and I'm an admin.

**Comment by dudaster** [comment]
Does it throw an error? Tevolution is free?

**Comment by tatumrae** [comment]
No, it doesn't throw any errors. It just says, "Sorry, you are not allowed to edit this item." When I try to click "Edit With Elementor" it just redirects me to the edit.php (posts) page.

Tevolution is a paid theme and plugin bundle.

**Comment by dudaster** [comment]
please make sure the all the loops template don't have any display conditions
